examples/release_highlights/plot_release_highlights_0_24_0.py

```diff
@@ -0,0 +1,225 @@
+# New PolynomialCountSketch kernel approximation function
+# -------------------------------------------------------
+# The new :class:`~sklearn.kernel_approximation.PolynomialCountSketch`
+# approximates a polynomial expansion of a feature space when used with linear
+# models, but uses much less memory than
+# :class:`~sklearn.preprocessing.PolynomialFeatures`, and is often faster than
+# using a polynomial kernel in SVM.
+
+from sklearn.datasets import fetch_covtype
+from sklearn.pipeline import make_pipeline
+from sklearn.model_selection import train_test_split
+from sklearn.kernel_approximation import PolynomialCountSketch
+from sklearn.svm import LinearSVC
+
+X, y = fetch_covtype(return_X_y=True)
+pipe = make_pipeline(PolynomialCountSketch(degree=4, n_components=1000),
+                     LinearSVC())
+X_train, X_test, y_train, y_test = train_test_split(X, y, train_size=5000,
+                                                    test_size=10000,
+                                                    random_state=42)
+pipe.fit(X_train, y_train)
+
+##############################################################################
+# Retrieving datasets from literature as pandas dataframes
+# --------------------------------------------------------
+# All the relevant functions loading datasets have a new parameter `as_frame`
+# allowing to load the dataset as a pandas DataFrame including columns with
+# appropriate dtypes (numeric, string, or categorical) and names.
+# The target is a pandas DataFrame or Series depending on the number of
+# `target_columns`.
+
+from sklearn.datasets import fetch_kddcup99
+
+df_kddcup99 = fetch_kddcup99(as_frame=True)
+df_kddcup99.target.head()
+
+##############################################################################
+# Individual Conditional Expectation
+# ----------------------------------
+# A new kind of partial dependence plot is available: the Individual
+# Conditional Expectation (ICE) plot. It visualizes the dependence of the
+# prediction on a feature for each sample separately with one line per sample.
+# See th :ref:`User Guide <individual_conditional>`
+
+from sklearn.linear_model import BayesianRidge
+from sklearn.datasets import fetch_california_housing
+from sklearn.inspection import plot_partial_dependence
+
+X, y = fetch_california_housing(return_X_y=True, as_frame=True)
+print('Computing partial dependence plots...')
+features = ['MedInc', 'AveOccup', 'HouseAge', 'AveRooms']
+est = BayesianRidge()
+est.fit(X, y)
+display = plot_partial_dependence(
+       est, X, features, kind="individual", subsample=50,
+       n_jobs=3, grid_resolution=20, random_state=0
+)
+display.figure_.suptitle(
+    'Partial dependence of house value on non-location features\n'
+    'for the California housing dataset, with BayesianRidge'
+)
+display.figure_.subplots_adjust(hspace=0.3)
+
+##############################################################################
+# New metrics available
```

Comment:
Not worth a highlight IMHO. But might be worth giving them visibility?

Reply:
The metrics module is one of the longer items in the changelog. I think that it is worth noticing. I'm keeping this section here.

Reply:
What do you mean by "longer items in the changelog"?

Reply:
Also, let's try to not unilaterally mark conversations as "resolved" please.

Reply:
> What do you mean by "longer items in the changelog"?

The metrics module contains 17 items  6 of which are new metrics.
I beg your pardon for my incorrect use of comparative in place of the superlative.

And I will resolve this comment because I think that 6 new metrics are worth noticing.

Reply:
Thanks for clarifying, that makes sense.

>  I beg your pardon for my incorrect use of comparative in place of the superlative.

I asked because I was genuinely confused. Perhaps I should have read more carefully the context of that section, I guess I would have gotten it. I'm not sure where this comment comes from, but please assume good intent from my part.

> And I will resolve this comment because I think that 6 new metrics are worth noticing.

I wish you hadn't resolved the first time. I would have liked to get input from other people, and resolving prevents others from commenting.

I personally only resolve if I can provide a fact, or once I know the other person has acknowledged my POV and is fine with it. When it's a matter of opinion, as it is here, it's healthier to leave the door open for discussions. An early resolving can be perceived as dismissing one's review, and this is what I felt there.